src/main/java/fr/xephi/authme/service/BukkitService.java

```diff
@@ -393,4 +393,11 @@ public BanEntry banIp(String ip, String reason, Date expires, String source) {
             return Optional.empty();
         }
     }
+
+    /**
+     * @return the IP string that this server is bound to, otherwise empty string
+     */
+    public String getIp() {
+        return Bukkit.getServer().getIp();
```

Comment:
Maybe this should be configurable in AuthMe to give it a more unique name that isn't a domain or IP address?

Reply:
Absolutely, that's a great point! Maybe in a further iteration so I can hopefully finish this PR before I leave for vacation, which is very soon.